packages/cli/src/ErrorReporting.ts

```diff
@@ -12,33 +12,65 @@ export const initErrorHandling = async () => {
 	});
 
 	const dsn = config.getEnv('diagnostics.config.sentry.dsn');
-	if (!config.getEnv('diagnostics.enabled') || !dsn) {
+	if (!dsn) {
 		initialized = true;
 		return;
 	}
 
 	// Collect longer stacktraces
 	Error.stackTraceLimit = 50;
 
-	const { N8N_VERSION: release, ENVIRONMENT: environment } = process.env;
+	const {
+		N8N_VERSION: release,
+		ENVIRONMENT: environment,
+		DEPLOYMENT_NAME: serverName,
+	} = process.env;
 
-	const { init, captureException, addGlobalEventProcessor } = await import('@sentry/node');
+	const { init, captureException, addEventProcessor } = await import('@sentry/node');
 
 	const { RewriteFrames } = await import('@sentry/integrations');
+	const { Integrations } = await import('@sentry/node');
 
+	const blockedIntegrations = [
+		'Console',
+		'Http',
+		'Undici',
+		'OnUncaughtException',
+		'LocalVariables',
+		'Context',
```

Comment:
Which one is this? Couldn't find it from [Sentry's docs](https://docs.sentry.io/platforms/node/configuration/integrations/default-integrations/?original_referrer=https%3A%2F%2Fwww.ecosia.org%2F). Only [ContextLines](https://docs.sentry.io/platforms/node/configuration/integrations/default-integrations/?original_referrer=https%3A%2F%2Fwww.ecosia.org%2F#contextlines)

Reply:
it's not documented from what I could tell. [It's the one that adds data about OS, Locale, Device, etc](https://github.com/getsentry/sentry-javascript/blob/develop/packages/node/src/integrations/context.ts). Which could be useful if we were to ever go back to tracking errors for self-hosters using NPM. For users on docker images, this provides no relevant data.